### PR TITLE
Breadcrumbs support for windows after the app restart/crash

### PR DESCRIPTION
**Pull request by konraddysput**
# Why

On the previous diff, we added a multi-session breadcrumbs support. In the past, we added also a similar solution for windows native dumps captured by the unity crash handler. In this diff, I tried to fix the flow for windows native and also simplify the code/remove redundant code. 

# testing strategy
1.  Crash app on Windows that will be captured by unity crash handler. See breadcrumbs on UI
2. Crash app on WIndows that will be captured by Backtrace. See breadcrumbs on UI
3. Crash app on Android - see breadcrumbs on UI

### CODE REVIEW

File: Runtime/BacktraceClient.cs [hunk 1]
```diff
@@ -1318,5 +1314,13 @@ private bool ShouldSkipReport(ReportFilterType type, Exception exception, string
                 || (SkipReport != null && SkipReport.Invoke(type, exception, message));
 
         }
+
+        internal IList<string> GetNativeAttachments()
+        {
+            return _clientReportAttachments.ToList()
```
Comment (perf2711) [inline Runtime/BacktraceClient.cs:1320]: is the first `.ToList()` here necessary?

File: Runtime/BacktraceDatabase.cs [hunk 2]
```diff
@@ -261,10 +264,39 @@ private void Start()
...
+            var isEnabled = Enable && Configuration.SendUnhandledGameCrashesOnGameStartup && isActiveAndEnabled;
+            var hasNativeConfiguration = _client && _client.NativeClient != null && _client.NativeClient is IStartupMinidumpSender;
+            if (isEnabled && hasNativeConfiguration)
+            {
+                var client = _client.NativeClient as IStartupMinidumpSender;
+                var attachments = _client.GetNativeAttachments();
+                if(string.IsNullOrEmpty(breadcrumbArchive))
+                {
+                    attachments.Add(breadcrumbArchive);
+                }
```
Comment (perf2711) [inline Runtime/BacktraceDatabase.cs:294]: is this valid? Do you want to add `breadcrumbArchive` only if it is `null` or empty?

**Review by vlussenburg — APPROVED** [review]

File: Runtime/BacktraceClient.cs (hunk 1 above)
Comment (konraddysput) [reply Runtime/BacktraceClient.cs]: you're right - we don't need it. Thank you.

File: Runtime/BacktraceDatabase.cs (hunk 2 above)
Comment (konraddysput) [reply Runtime/BacktraceDatabase.cs]: I messed this up before finalizing pull request. Now it should work. Thank you!

**Review by perf2711 — APPROVED** [review]

**Pull request merged**